packages/@blockprotocol/graph/src/graph-service.json

```diff
@@ -0,0 +1,688 @@
+          }
+        },
+        "required": ["entity"]
+      },
+      "errorsCodes": ["FORBIDDEN", "INVALID_INPUT", "NOT_FOUND"]
+    },
+    {
+      "name": "deleteEntity",
+      "description": "Request to delete an entity, expecting a boolean in response.",
+      "source": "block",
+      "respondedToByMessage": "deleteEntityResponse",
+      "sentOnInitialization": false,
+      "data": {
+        "type": "object",
+        "properties": {
+          "entityId": {
+            "description": "The entityId of the entity to delete",
+            "type": "string"
+          }
+        },
+        "required": ["entityId"]
+      },
+      "errorCodes": []
+    },
+    {
+      "name": "deleteEntityResponse",
+      "description": "The response to a request to delete an entity",
+      "source": "embedder",
+      "respondedToByMessage": null,
+      "sentOnInitialization": false,
+      "data": {
+        "type": "boolean"
+      },
+      "errorsCodes": ["FORBIDDEN", "INVALID_INPUT", "NOT_FOUND"]
+    },
+    {
+      "name": "getEntity",
+      "description": "Request to retrieve an entity, expecting the entity in response.",
+      "source": "block",
+      "respondedToByMessage": "getEntityResponse",
+      "sentOnInitialization": false,
+      "data": {
+        "type": "object",
+        "properties": {
+          "entityId": {
+            "description": "The entityId of the entity to retrieve",
+            "type": "string"
+          }
+        },
+        "required": ["entityId"]
+      },
+      "errorCodes": []
+    },
+    {
+      "name": "getEntityResponse",
+      "description": "The response to a request to get an entity",
+      "source": "embedder",
+      "respondedToByMessage": null,
+      "sentOnInitialization": false,
+      "data": {
+        "type": "object",
+        "properties": {
+          "entity": {
+            "$ref": "https://blockprotocol.org/types/Entity"
+          }
+        },
+        "required": ["entity"]
+      },
+      "errorsCodes": ["FORBIDDEN", "INVALID_INPUT", "NOT_FOUND"]
+    },
+    {
+      "name": "aggregateEntities",
+      "description": "Request to retrieve an aggregation of entities.",
+      "source": "block",
+      "respondedToByMessage": "aggregateEntitiesResponse",
+      "sentOnInitialization": false,
+      "data": {
+        "type": "object",
+        "properties": {
+          "operation": {
+            "$ref": "https://blockprotocol.org/types/LinkedAggregationOperation"
+          }
+        },
+        "required": ["operation"]
+      },
+      "errorCodes": []
+    },
+    {
+      "name": "aggregateEntitiesResponse",
+      "description": "The response to a request to retrieve an aggregation of entities",
+      "source": "embedder",
+      "respondedToByMessage": null,
+      "sentOnInitialization": false,
+      "data": {
+        "type": "object",
+        "properties": {
+          "operation": { "type": "object" },
```

Comment:
Is this operation also a `{ "$ref": "https://blockprotocol.org/types/LinkedAggregationOperation" }`?

Reply:
It is, thank you